include/swift/ClangImporter/ClangImporter.h

```diff
@@ -73,8 +90,11 @@ class ClangImporter final : public ClangModuleLoader {
   Implementation &Impl;
 
   ClangImporter(ASTContext &ctx, const ClangImporterOptions &clangImporterOpts,
-                DependencyTracker *tracker);
+                DependencyTracker *tracker,
+                std::unique_ptr<DWARFImporterDelegate> dwarfImporterDelegate);
 
+  ModuleDecl *loadModuleClang(SourceLoc importLoc,
```

Comment:
This name is no longer contrastive anymore. Can you rename it and/or add a comment about how this is different from `loadModule`?

Reply:
https://github.com/apple/swift/pull/26682